### ISSUE TITLE
Bug: false trigger on valid imports

### ISSUE THREAD

**Issue by AlexandrHoroshih** [feature-sliced/eslint-config#60]
I have an app with `react, effector, patronum, antd` and bunch of other stuff

After i had added `@feature-sliced/eslint-config`, errors like this started to appear:
```ts
  2:26  error  Reaching to "effector-react/ssr" is not allowed  import/no-internal-modules
  3:23  error  Reaching to "patronum/delay" is not allowed      import/no-internal-modules
```
The problem is that imports like that are perfectly valid and intended by these libraries 🤷‍♂️ 
I suggest not to run `import/no-internal-modules` rule against imports from node_modules (or against non-feature-sliced paths at all)

Small reproduce: https://stackblitz.com/edit/fs-eslint-false-trigger?file=src/app/index.js
Run `npm run lint` in the terminal to see the issue

**Comment by azinit** [comment]
**Thanks for issue!**

For a while you can customize your [boundaries manually](https://github.com/feature-sliced/eslint-config#customization):

```js
{
  "extends": [
    "@feature-sliced/eslint-config/rules/import-order",
    "@feature-sliced/eslint-config/rules/layers-slices", 
    // Related with bug =>  "@feature-sliced/eslint-config/rules/public-api",
  ]
}
```

**Comment by azinit** [comment]
@Krakazybik SLA 40 MIN, KARL! 🚀 

![image](https://user-images.githubusercontent.com/42924400/147464341-8db5dc16-da10-4661-9a27-72f31f66cd15.png)

**Comment by azinit** [comment]
@AlexandrHoroshih JFYI, I've published new patch version - `0.1.0-alpha.1`

You can try to use it with this bugfix

**Comment by azinit** [comment]
If problem remained, reopen issue ✊
